iceberg/iceberg-rest-server/src/test/java/org/apache/gravitino/iceberg/service/TestIcebergCatalogWrapperManagerForREST.java

```diff
@@ -55,7 +55,7 @@ public void testValidGetOps(String rawPrefix) {
   }
 
   @ParameterizedTest
-  @ValueSource(strings = {"hello", "\\\n\t\\\'", "\u0024", "\100", "[_~", "default_catalog/"})
+  @ValueSource(strings = {"hello", "\\\n\t\\\'", "\u0024", "\100", "[_~"})
```

Comment:
Why do we do such change?

Reply:
`default_catalog` is the default catalog name if the user doesn't specify the catalog name in `prefix`.  After this PR , we add `ENDPOINTS` to the config interface which represent the capacity of the underlying catalog backend.  if user doesn't specifing the catalog name, we will response the catalog capacibilities from `default_catalog` and set `prefix` to the catalog name of `default_catalog`.  so `default_catalog` is a valid catalog identifier.